package.json

```diff
@@ -7,7 +7,7 @@
     "format": "prettier --write \"**/*.{js,md,yml,ts,json,yaml}\"",
     "lint": "prettier --list-different \"**/*.{js,md,yml,ts,json,yaml}\"",
     "clean": "lerna run --parallel clean",
-    "build": "lerna run --parallel build",
+    "build": "lerna run build",
```

Comment:
Isn't this needed?

Reply:
Ran in parallel throws an error in one of the big query packages.  It makes builds quicker, but it isn't essential. It's a strange error as it's trying to compile before the `tsc` binary is installed I think. It might be a problem with the structure of the packages (i.e packages within packages).